packages/unmock-runner/package.json

```diff
@@ -14,5 +14,8 @@
   },
   "dependencies": {
     "unmock-core": "file:../unmock-core"
+  },
+  "devDependencies": {
+    "unmock-jest-runner": "0.0.0"
```

Comment:
Hmm I'm not sure if packages should add their own devdependencies like this, I think we've usually kept them at the root level `package.json`. One never runs tests for a single package but for the whole Lerna project. Could this maybe be removed and added to root-level dev dependencies?

Reply:
Ah that's a good point about lerna that I hadn't considered. Thank you! I can update 😊